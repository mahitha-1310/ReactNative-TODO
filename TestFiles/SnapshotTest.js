import React from "react";
import { Text, View } from "react-native";
import { StyleSheet } from "react-native";

function SnapshotTest(props) {
  return (
    <View style={styles.container}>
      <Text style={styles.text}>Hiiiiiiiii</Text>
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flexDirection: "row",
    flex: 1,
    alignItems: "center",
    justifyContent: "center",
  },
  text: {
    color: "black",
    fontWeight: "bold",
  },
});

export default SnapshotTest;
