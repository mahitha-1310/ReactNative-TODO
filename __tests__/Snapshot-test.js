import React from "react";
import SnapshotTest from "../TestFiles/SnapshotTest";
import renderer from "react-test-renderer";

test("Test Snapshot", () => {
  const snap = renderer.create(<SnapshotTest />).toJSON();
  expect(snap).toMatchSnapshot();
});

// import React from "react";
// import App from "../App";
// import { StateProvider } from "../store/StateContext";
// import renderer from "react-test-renderer";

// test("State Provider snapshot", () => {
//   const snap = renderer.create(<StateProvider />).toJSON;
//   expect(snap).toMatchSnapshot();
// });
