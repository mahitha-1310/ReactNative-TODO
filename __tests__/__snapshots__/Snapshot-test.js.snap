// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Snapshot 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      {
        "color": "black",
        "fontWeight": "bold",
      }
    }
  >
    Hiiiiiiiii
  </Text>
</View>
`;
